settings.py

```diff
@@ -715,6 +714,10 @@ def JINJA_CONFIG():
             'js/profile.js',
             'js/moz-jquery-plugins.js',
         ),
+        'document': (
```

Comment:
Stephanie and I noticed we'd both written JS features (her: social, me: helpfulness) that should (eventually) appear on all document pages, but not on editing pages. We agreed this might be a general case for scripts in the future.